odl/discr/discretization.py

```diff
@@ -241,40 +225,97 @@ def domain(self):
         """Domain of the continuous space."""
         return self.uspace.domain
 
+    # Pass-through attributes of the wrapped ``dspace``
+    def zero(self):
+        """Return the element of all zeros."""
+        return self.element_type(self, self.dspace.zero())
+
+    def one(self):
+        """Return the element of all ones."""
+        return self.element_type(self, self.dspace.one())
+
+    @property
+    def weighting(self):
+        """This space's weighting scheme."""
+        return getattr(self.dspace, 'weighting', None)
```

Comment:
The `is_weighted` thing too? Feels like that one should rather return `False` in that case.